### ISSUE TITLE
cert has expired

### ISSUE THREAD

**Issue by Chr1sDub** [terenc3/MMM-Tankerkoenig#5]
Hey there
I am encountering an issue with the tankerkoenig module and can´t find any know issues about this. Could use some help. Thank you!

 > 1|mm  | [13.10.2021 22:02.15.800] [LOG]
> 1|mm  | FetchError: request to https://creativecommons.tankerkoenig.de/json/list.php?lat=50.0191354&lng=10.2822101&rad=10&type=all&apikey=9575b8bd-1fda-c5cf-f22e-c288a1e0d9a3&sort=dist failed, reason: certificate has expired
> 1|mm  |     at ClientRequest.<anonymous> (/home/pi/MagicMirror/modules/MMM-Fuel/node_modules/node-fetch/lib/index.js:1483:11)
> 1|mm  |     at ClientRequest.emit (events.js:315:20)
> 1|mm  |     at TLSSocket.socketErrorListener (_http_client.js:426:9)
> 1|mm  |     at TLSSocket.emit (events.js:315:20)
> 1|mm  |     at emitErrorNT (internal/streams/destroy.js:92:8)
> 1|mm  |     at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
> 1|mm  |     at processTicksAndRejections (internal/process/task_queues.js:84:21) {
> 1|mm  |   type: 'system',
> 1|mm  |   errno: 'CERT_HAS_EXPIRED',
> 1|mm  |   code: `'CERT_HAS_EXPIRED'`

**Comment by terenc3** [comment]
@Chr1sDub Please do not redact your initial message, otherwise it is hard to follow what happened. Mistakes can occur, thats why i'm using a issue tracker :)

> Hey there
I am encountering an issue with the tankerkoenig module and can´t find any know issues about this. Could use some help. Thank you!

    1|mm | [13.10.2021 22:02.15.800] [LOG]
    1|mm | FetchError: request to https://creativecommons.tankerkoenig.de/json/list.php?lat=50.0191354&lng=10.2822101&rad=10&type=all&apikey=9575b8bd-1fda-c5cf-f22e-c288a1e0d9a3&sort=dist failed, reason: certificate has expired
    1|mm | at ClientRequest. (/home/pi/MagicMirror/modules/MMM-Fuel/node_modules/node-fetch/lib/index.js:1483:11)
    1|mm | at ClientRequest.emit (events.js:315:20)
    1|mm | at TLSSocket.socketErrorListener (_http_client.js:426:9)
    1|mm | at TLSSocket.emit (events.js:315:20)
    1|mm | at emitErrorNT (internal/streams/destroy.js:92:8)
    1|mm | at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    1|mm | at processTicksAndRejections (internal/process/task_queues.js:84:21) {
    1|mm | type: 'system',
    1|mm | errno: 'CERT_HAS_EXPIRED',
    1|mm | code: 'CERT_HAS_EXPIRED'

First you confused my module MMM-Tankerkönig with MMM-Fuel. Furthermore this is a issue related to ssl certificates and related to MagicMirror itselfs (see MichMich/MagicMirror#2671). I may check how my module is affected, but recommend from a gut feeling to upgrade MagicMirror to its latest version. For workarounds regarding MMM-Fuel check out fewieden/MMM-Fuel#26
